src/components/collaboration-mode/collaboration-mode-picker.tsx

```diff
@@ -0,0 +1,101 @@
+import { Button, Popover, Icon } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { paragraph } from '@wordpress/icons';
+
+import { viewIcon } from './view-icon';
+
+import '@/components/collaboration-mode/collaboration-mode-picker.scss';
+
+type CollaborationModeType = 'view' | 'edit';
```

Comment:
Could this be turned into an enum that's exported, and re-usable outside this component? We are going to be implementing the functionality and it'd be nice to prep for that.